.github/workflows/build.yml

```diff
@@ -75,7 +75,7 @@ jobs:
           path: server/engine/target/native/agent-output/test/*
       - name: Build Server and Dialects Jars
         working-directory: server
-        run: mvn clean package --no-transfer-progress
+        run: mvn clean -DskipTests package --no-transfer-progress
```

Comment:
```
Warning: No files were found with the provided path: server/engine/target/perf.csv. No artifacts will be uploaded.
```
We collect performance data from tests.

Reply:
Ok, looks like we should collect them on the previous stages

Reply:
Sure, any time its possible, just don't loose them :)